src/date-picker/date-picker-dialog.jsx

```diff
@@ -41,18 +42,6 @@ const DatePickerDialog = React.createClass({
     muiTheme: React.PropTypes.object,
   },
 
-  getDefaultProps: function() {
```

Comment:
Oh, I hadn't realised. Well, I guess it's harmless having the defaults repeated in date-picker-dialog. I'll put them back.